### DISCUSSION TITLE
I'm getting error when want to make admin panel like a SPA application

### DISCUSSION THREAD

**Discussion by CodeWithPranta** [filamentphp/filament#8666] [General]
I read documentation and in configuration section I saw ->spa() method can make this application like SPA. But getting this error after applied. 
    ___Method Filament\Panel::spa does not exist._

**Comment by leandrocfe** [comment]
Filament version?

**Reply by CodeWithPranta** [reply]
"filament/filament": "3.0-stable",

**Reply by leandrocfe** [reply]
`composer update` ?

**Comment by CodeWithPranta** [comment]
Everything is updated, using liverwire 3.0.5

**Comment by CodeWithPranta** [comment]
Thanks, problem solved after reinstall the application using latest version composer.
